.gitignore

```diff
@@ -61,3 +61,5 @@ DEBUG_PROFILE.zip
 packaging/add_itemsize/*
 !packaging/add_itemsize/add_itemsize.js
 /.idea/AIAssistantCustomInstructionsStorage.xml
+
+config/
```

Comment:
what's this for?

Reply:
Oh that's a directory I had made for mounting the container in while testing. I can remove that line if needed